src/lib/components/particles/waffle/index.jsx

```diff
@@ -0,0 +1,37 @@
+import defaultStyles from "./style.module.css"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+
+const renderUnit = (squares, attributes) => squares ? <rect {...attributes} /> : <circle {...attributes} />
+
+export const Waffle = ({ units, rows, total, squares = false, abbreviationAccessor = 'party', onMouseOver, paddingTop, showHalfLine, styles }) => {
```

Comment:
And rather than a boolean values for `squares`, I would suggest an enum approximation (since enums don't exist in Javascript), e.g.

```js
export const WaffleType = {
  circle: 'circle',
  square: 'square',
}

export const Waffle = ({ type = WaffleType.circle }) => { }
```